src/Logout.tsx

```diff
@@ -19,6 +25,14 @@ const Logout: FC<Props> = ({history}) => {
 
   useEffect(() => {
     const handleSignOut = async () => {
+      if (CLOUD && isFlagEnabled('authSessionCookieOn')) {
+        const url = new URL(
+          `${window.location.origin}${CLOUD_SIGNOUT_PATHNAME}`
+        )
+        window.location.href = url.href
+        return
+      }
+
       if (CLOUD) {
         window.location.href = `${CLOUD_URL}${CLOUD_LOGOUT_PATH}`
         return
```

Comment:
```suggestion
      if (CLOUD) {
       const url = isFlagEnabled('authSessionCookieOn')
          ? `${window.location.origin}${CLOUD_SIGNOUT_PATHNAME}`
          : `${CLOUD_URL}${CLOUD_LOGOUT_PATH}`
        
        window.location.href = url
        return
      }
```

or something like that.  #optional